### DISCUSSION TITLE
Hole in hammerhead texture

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#14638] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

You can't unsee it

![image](https://github.com/user-attachments/assets/8e1debe6-5dee-44ab-b15a-68c1eb53f3f1)

### Reproduction steps

Zoom in

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.5.9.2 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Fixed in https://github.com/FakeFishGames/Barotrauma-development/commit/1cc825a089133c94626e5b61a9f75954ca1e6f53
